infrastructure/conf/nginx-conf-local/default

```diff
@@ -69,7 +69,7 @@ server {
         include fastcgi_params;
         fastcgi_param SCRIPT_FILENAME /home/site/wwwroot/api/public/index.php;
     }
-    location = /graphql-playground {
+    location = /graphiql {
```

Comment:
If switching to `graphiql`, there are a few references that should also be changed:
https://github.com/GCTC-NTGC/gc-digital-talent/blob/594d5cc6de52ea7ffc4d58799f438f703ab408f4/api/README.md?plain=1#L12
https://github.com/GCTC-NTGC/gc-digital-talent/blob/594d5cc6de52ea7ffc4d58799f438f703ab408f4/infrastructure/conf/xdebug.ini#L9

Reply:
Good catch!
[update documentation for graphiql](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5869/commits/ff2e62ad247cda11218c8acfd3b4e2565ce9fb11)